tests/core/test_snapshot.py

```diff
@@ -300,9 +315,19 @@ def test_lookback(snapshot: Snapshot, make_snapshot):
     assert snapshot.missing_intervals("2023-01-28", "2023-01-28", "2023-01-30 05:00:00") == [
         (to_timestamp("2023-01-28"), to_timestamp("2023-01-29")),
     ]
+    assert snapshot.missing_intervals(
+        "2023-01-28", "2023-01-28", "2023-01-30 05:00:00", allow_partial=True
+    ) == [
+        (to_timestamp("2023-01-28"), to_timestamp("2023-01-29")),
```

Comment:
Shouldn't this be `(to_timestamp("2023-01-28"), to_timestamp("2023-01-30"))`?

Reply:
Why though? The end was set to `2023-01-28` + partial and we get `2023-01-29`

Reply:
Yeah sorry get it now.